tests/functional/test_reinstall_overrides.py

```diff
@@ -58,7 +58,7 @@ async def test_reinstall_overrides(
         opts={'rose_template_vars': ['var="CLIinstall"']})
 
     # Play workflow
-    run_ok(f'cylc play --pause {wid}')
+    run_ok(f'cylc play --pause {wid} --host localhost')
```

Comment:
Unrelated to PR, but this was causing flakiness becuase the test becoms time-sensitve when running cylc play on a remote host.